mobile/src/components/uam/uam.tsx

```diff
@@ -11,11 +12,20 @@ import { UsersTable } from './users-table/users-table';
 
 const UAM: FC = () => {
   const navigate = useAppNavigate();
+  const dispatch = useAppDispatch();
 
   const handleGroupCreate = (): void => {
     navigate.navigate(AppScreenName.UAM_GROUPS_CREATE);
   };
 
+  useEffect(() => {
+    const timer = setInterval(() => {
+      dispatch(authActions.loadCurrentUser());
+    }, 2000);
+
+    return () => clearInterval(timer);
+  }, [dispatch]);
```

Comment:
not a good approach, we need to check if user changed himself and then reload current user

Reply:
Oh...it's about when a user deletes a group with yourself?